src/database/services/keycloak-api-service.ts

```diff
@@ -1,12 +1,12 @@
-import type { User } from "../schemas/keycloak/user";
+import type { User } from "../../types/keycloak/user";
 import fetch from "node-fetch";
 
 /**
  * Interface for a KeycloakApiService.
  */
 export interface KeycloakApiService {
   getUsers: () => Promise<User[]>;
-  findUser: (id: string) => Promise<User[]>;
+  findUser: (id: string) => Promise<User>;
 }
 
```

Comment:
Maybe we can won't need this empty space as Josh reviewed our old PR and commented about this